auth/auth_test.go

```diff
@@ -153,6 +161,12 @@ func TestRevertedTxNonceMiddleware(t *testing.T) {
 	})
 	require.NoError(t, err)
 
+	nonceTxBytes3, err := proto.Marshal(&NonceTx{
```

Comment:
Move this down, closer to where it's actually used.

Reply:
Fixed